pkg/rpc/server_test.go

```diff
@@ -132,15 +132,6 @@ func TestRPC(t *testing.T) {
 		assert.Equal(t, "Invalid assetid", res.Result)
 	})
 
-	t.Run("getaccountstate_positive", func(t *testing.T) {
-		rpc := `{"jsonrpc": "2.0", "id": 1, "method": "getaccountstate", "params": ["a21e4f7178607089e4fe9fab1300d1f5a3d348be"]}`
```

Comment:
Sure it depends on the data that was in the LevelDB. It's not good losing testcases.

Reply:
True, working on it